server/api/handler/admin/user.go

```diff
@@ -0,0 +1,117 @@
+package admin
+
+import (
+	"github.com/gobuffalo/pop/v6"
+	"github.com/gofrs/uuid"
+	"github.com/labstack/echo/v4"
+	"github.com/teamhanko/passkey-server/api/helper"
+	"github.com/teamhanko/passkey-server/api/services/admin"
+	"github.com/teamhanko/passkey-server/persistence"
+	"net/http"
+)
+
+type UserHandler interface {
+	List(ctx echo.Context) error
+	Get(ctx echo.Context) error
+	Remove(ctx echo.Context) error
+}
+
+type userHandler struct {
+	persister persistence.Persister
+}
+
+func NewUserHandler(persister persistence.Persister) UserHandler {
+	return &userHandler{persister: persister}
+}
+
+func (uh *userHandler) List(ctx echo.Context) error {
```

Comment:
Would be nice to have pagination, because the list could be long in the future.

Reply:
I also added the pagination params to the admin spec